src/auth.ts

```
@@ -28,16 +29,146 @@ export const authConfig = {
     ...(env.RESEND_API_KEY
       ? [ResendProvider({ apiKey: env.RESEND_API_KEY, from: env.EMAIL_FROM })]
       : []),
+    CredentialsProvider({
+      id: "otp",
+      name: "Email OTP",
+      credentials: {
+        email: { label: "Email", type: "email" },
+        code: { label: "Code", type: "text" },
+      },
+      async authorize(credentials) {
```

ESLint

[error] Expected object keys to be in natural insensitive ascending order. 'authorize' should be before 'credentials'.

(sort-keys-fix/sort-keys-fix)